extjsdk/src/main/java/io/vantiq/extjsdk/ExtensionWebSocketClient.java

```diff
@@ -287,7 +316,25 @@ public CompletableFuture<Boolean> initiateFullConnection(String url, String toke
         authenticate(token);
         return connectToSource();
     }
-
+    
+    /**
```

Comment:
Allows passing in of more detail.  Useful in cases where the connector is separately configured via other means.